src/main/java/com/icthh/xm/uaa/security/DomainJwtAccessTokenDetailsPostProcessor.java

```diff
@@ -6,14 +6,16 @@
 import com.icthh.xm.commons.lep.spring.LepService;
 import java.util.HashMap;
 import java.util.Map;
+
+import com.icthh.xm.uaa.lep.keyresolver.OptionalProfileHeaderResolver;
 import org.springframework.security.oauth2.provider.OAuth2Authentication;
 import org.springframework.stereotype.Component;
 
 @Component
 @LepService(group = "security")
 public class DomainJwtAccessTokenDetailsPostProcessor {
 
-    @LogicExtensionPoint("ProcessJwtAccessTokenDetails")
+    @LogicExtensionPoint(value = "ProcessJwtAccessTokenDetails", resolver = OptionalProfileHeaderResolver.class)
```

Comment:
WARNING!!! KEEP backward compatibility!!!

Reply:
Backward compatibility is present. New test was added to check lep function with profile header. And we already have existing test to check lep function without profile header